File: beginner/pagina02/1044/1044.php
<?php 

    list($a, $b) = explode(" ", readline());

    $a = (int) $a;
    $b = (int) $b;

    if($a < $b){
        $temp = $b;
        $b = $a;
        $a = $temp;
    }

    if($a % $b == 0){
        echo "Sao Multiplos\n";
    } else {

        echo "Nao sao Multiplos\n";
    }
?>

File: beginner/__1000__/1044/1044.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input).unwrap();

    let data: Vec<u32> = input.trim().split_whitespace()
        .map(|x| x.parse().unwrap())
        .collect();

    let (a, b) = (data[0], data[1]);

    if a == 0 || b == 0 {
        println!("Nao sao Multiplos");
    } else if b % a == 0 || a % b == 0 {
        println!("Sao Multiplos");
    } else {
        println!("Nao sao Multiplos");
    }
}